Userland/Libraries/LibWeb/FileAPI/File.cpp

```diff
@@ -77,4 +87,53 @@ WebIDL::ExceptionOr<JS::NonnullGCPtr<File>> File::construct_impl(JS::Realm& real
     return create(realm, file_bits, file_name, options);
 }
 
+WebIDL::ExceptionOr<void> File::serialization_steps(HTML::SerializationRecord& record, bool)
+{
+    auto& vm = this->vm();
+
+    record.append(HTML::ValueTag::FileObject);
```

Comment:
Similar to my other comment, I think we should add `ValueTag::Serializable` and serialize a `interface_name()` function that we add to the Serializable interface inside structured_serialize() instead of making it the job of the Serialization steps of each interface to serialize this implementation detail of our structured serialize implementation.

Reply:
Yes, that's a nice idea!